src/lib/components/select/namespace-select.svelte

```diff
@@ -11,7 +11,7 @@
     (namespace) => namespace.namespaceInfo.name,
   );
 
-  let selectedNamespace = $currentNamespace;
+  $: selectedNamespace = $currentNamespace;
```

Comment:
What is this doing? The `$currentNamespace` should be reactive already.

Reply:
Without
https://user-images.githubusercontent.com/1903839/157760417-3d0ada05-fc0a-4eaa-9c75-49b85370bfc0.mov

With

https://user-images.githubusercontent.com/1903839/157760522-baf493d2-ff82-477a-b95d-519548c51cfa.mov

Reply:
For whatever reason when browsing between different "current" namespaces it's not reacting to the changes and that enabled it to switch in the dropdown.